app/javascript/packs/components/FileOption.jsx

```diff
@@ -21,24 +21,40 @@ class FileOption extends React.Component {
     return this.props.answer && this.props.answer.attachment_file_name;
   }
 
+  isUploading() {
+    return this.props.answer && this.props.answer.fetchStatus === FETCH_IN_PROGRESS;
+  }
+
+  downloadFile() {
```

Comment:
Unused function: leftover from trying to use `browser.downloads`?

Reply:
Oops yeah